tests/test_sshfsmount.cpp

```diff
@@ -350,17 +429,11 @@ TEST_F(SshfsMount, throws_when_unable_to_get_current_dir)
 TEST_F(SshfsMount, executes_commands)
 {
     CommandVector commands = {
-        {"which sshfs", "/usr/bin/sshfs"},
         {"pwd", "/home/ubuntu"},
```

Comment:
```suggestion
```

Reply:
Ok...I *thought* we could make it a default command, but it was added in #1359, so I left it as a tested command.  But since you mentioned it, then we'll do it :smile: